src/sentry/static/sentry/app/plugins/defaultIssuePlugin.jsx

```diff
@@ -1,14 +0,0 @@
-import React from 'react';
-
-import BasePlugin from 'app/plugins/basePlugin';
-import IssueActions from 'app/plugins/components/issueActions';
-
-export class DefaultIssuePlugin extends BasePlugin {
-  renderGroupActions(props) {
-    return <IssueActions plugin={this} {...props} />;
-  }
-}
-
-DefaultIssuePlugin.DefaultIssueActions = IssueActions;
```

Comment:
Will removing this break anything?

Reply:
@billyvg not with the refactor that I did as Mark points out

Reply:
@scefali Are these modules used for external/custom plugins? could we break those?

Reply:
@billyvg I searched on Github and I don't think anyone is hooking into plugins this way. Most plugins don't use these modules. I would be surprised if any community-based plugins use this. I don't think it's something we should support long term as well.

Reply:
By most, I mean all but Jira and SessionStack don't use this hook